contributing/development/compiling/compiling_with_mono.rst

```diff
@@ -132,37 +86,32 @@ Example (Windows)
 
     # Build editor binary
     scons p=windows target=editor module_mono_enabled=yes
+    # Build export templates
+    scons p=windows target=template_debug module_mono_enabled=yes
+    scons p=windows target=template_release module_mono_enabled=yes
+    
     # Generate glue sources
-    bin\godot.windows.editor.x86_64.mono --generate-mono-glue modules/mono/glue
+    bin/godot.windows.editor.x86_64.mono --generate-mono-glue modules/mono/glue
```

Comment:
This may have been intentional since these are Windows instructions. Even if Windows supports `/`, we may want to use whatever is preferred by Windows, but I don't have a strong opinion about it.

Reply:
I changed it mostly because it messed up the syntax highlighting and that was the only backslash on the page.